source/mainnet/smart-contracts/guides/compile-module.rst

```diff
@@ -30,14 +30,29 @@ To help building smart contract modules and to take advantage of features
 such as :ref:`contract schemas <contract-schema>`, Concordium recommends using the
 ``cargo-concordium`` tool for building Rust_ smart contracts.
 
-In order to build a smart contract, run:
+In order to build a smart contract for deployment, run:
 
 .. code-block:: console
 
-   $cargo concordium build
+   $cargo concordium build --verifiable docker.io/concordium/verifiable-sc:1.70.0 --out contract.wasm.v1
 
 This uses Cargo_ for building, but runs further optimizations on the result.
 
+The ``--verifiable`` option  is available on ``cargo-concordium`` version 3.1 or newer. It will cause ``cargo-concordium`` to build the sources
+in a fixed container to make the build reproducible. It will additionally produce
+a ``tar`` file with packaged sources that were used for the build. The list of
+available docker images can be found on `DockerHub. <https://hub.docker.com/r/concordium/verifiable-sc>`_
+
+The ``tar`` archive should be uploaded and made publicly available, and its
+link should be embedded into the deployed module using the ``cargo concordium edit-build-info``
+command.
+
+The ``--out`` option is required for verifiable builds, and instructs ``cargo-concordium`` to output the build artifact at the specified location.
+
+Verifiable builds require a container runtime such as `Docker <https://www.docker.com/>`_ to be installed.
```

Comment:
This should probably also be mentioned in the `setup-tools` guide, that we link in the preparation section

Reply:
Added